chsdi/lib/translator.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+import os
+import psycopg2
+from cachetools import TTLCache
+from psycopg2.extras import RealDictCursor
+from psycopg2.extensions import register_type, UNICODE
+
+CACHE_TTL = 3600  # Cache duration in seconds --> 1 hour cache for translations
+CACHE_SIZE = 10000
+
+
+class Translator:
+
+    _translations = None
+    _supported_languages = ['en', 'fr', 'de', 'rm', 'it']
+
+    @staticmethod
+    def get_db_connection():
+        return psycopg2.connect(host=os.environ.get('DBHOST'), dbname='bod_master', user='www-data')
```

Comment:
please use the env variable
https://github.com/geoadmin/mf-chsdi3/blob/ltkum_dynamic_translation/rc_dev#L7
for the database staging

Reply:
It has been done.